Formula/openimageio.rb

```diff
@@ -0,0 +1,79 @@
+class Openimageio < Formula
+  desc "Library for reading, processing and writing images"
+  homepage "http://openimageio.org/"
+  url "https://github.com/OpenImageIO/oiio/archive/Release-1.8.7.tar.gz"
+  sha256 "ed23ecdd3e4037a411c69a6a681273d01d4a2fbcc223eac86daba77dd03fc5ed"
+  head "https://github.com/OpenImageIO/oiio.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+  depends_on "boost-python"
+  depends_on "boost-python3"
+  depends_on "ffmpeg"
+  depends_on "freetype"
+  depends_on "giflib"
+  depends_on "ilmbase"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libraw"
+  depends_on "libtiff"
+  depends_on "opencolorio"
+  depends_on "openexr"
+  depends_on "python3"
+  depends_on "webp"
+
+  def install
+    # -DUSE_OPENSSL=OFF can be removed in 1.9, see
+    # https://github.com/Homebrew/homebrew-core/pull/22522#issuecomment-364831533
+    args = std_cmake_args + %w[
+      -DEMBEDPLUGINS=ON
+      -DSELF_CONTAINED_INSTALL_TREE=ON
```

Comment:
I think this option isn't being used.
```
CMake Warning:
  Manually-specified variables were not used by the project:

    HAVE_CLOCK_GETTIME
    SELF_CONTAINED_INSTALL_TREE
```

Reply:
Yes, I saw that too. Still wondering why it's in the Makefile and not getting used.

Reply:
It looks like it was removed in https://github.com/OpenImageIO/oiio/pull/1760 but not removed from the Makefile.

Reply:
Yes, it doesn't appear elsewhere, so I'll remove it. @lgritz: You may also want to consider removing `-DSELF_CONTAINED_INSTALL_TREE` at https://github.com/OpenImageIO/oiio/blob/567ba9a217120b2315147d56de76ffa8593f355f/Makefile#L30

Reply:
> [...] so I'll remove it.

Done